two_points_on_the_same_branch.sh

```diff
@@ -203,9 +210,9 @@ create_opam() {
 
     if [ ! -z "$BENCH_DEBUG" ]; then echo "DEBUG: $1_coq_commit_long = $COQ_HASH_LONG"; fi
 
-    if opam install coq.$coq_opam_version -b -j$number_of_processors; then
-        :
-    else
+    opam install -vvv -b -j$number_of_processors coq.$coq_opam_version
+
+    if [ ! $? -eq 0 ]; then
```

Comment:
Won't set -e break this? Why not do `if ! opam install ...`?

Reply:
Yes, sorry for the noise; I am just trying to debug why opam is not calling `make -j 2` for Coq.